study-group/Toast-Implementation-Studies.md

```diff
@@ -90,9 +90,151 @@ The `restoreFocusOnClear` attribute restores page's focus once the toast is clea
 
 *TODO: Fill in study for this library*
 
-## ngx-toastr
+## [ngx-toastr](https://github.com/scttcper/ngx-toastr)
+ngx-toastr is a library from author [@scttcper](https://github.com/scttcper) to create a toast notification built for Angular.
 
-*TODO: Fill in study for this library*
+
+### Design & Sample Code
+Before using ngx-toastr, setup is required. First, add the CSS:
+
+```js
```

Comment:
I think this is CSS, not JS. In which case the comments need to be `/* ... */`.

Reply:
Yeah, just got in the habit of writing js, thanks for catching that